### ISSUE TITLE
Render is not working

### ISSUE THREAD

**Issue by nkiransingh** [zippyui/react-datagrid#164]
Hi,
In my datagrid, i have custom TextRender as column render with ADD row and delete row button outside of grid.When i have 5 rows and deleted the second row and row getting deleted when i am trying to update the value it is not updated :(. Can someone please help me. when i am reassigning the list to datagrid looks like using old render is been used and not creating a new render again

**Comment by manisht86** [comment]
That's right, I'm facing a similar issue. If you have a custom renderer on your column, the renderer's render is not getting called again with new state. For e.g. if I enter a value in a TextInputRenderer on my column, that value is retained even when the data provider of the grid get's changed by the parent/holding component. Seems like an issue. Not sure if someone is facing the same issue with renderers.

**Comment by vedran-maestral** [comment]
Same issue here. Custom render is not being called for some reason

**Comment by vedran-maestral** [comment]
Is there a solution to this issue?

**Comment by scheung38** [comment]
so has anyone managed to get a delete row button working? Please share, much appreciated.
